src/games/elements/ElementNav.vue

```diff
@@ -0,0 +1,63 @@
+<script>
+export default {
+  name: 'ElementNav',
+  props: ['game', 'elements'],
+  data () {
+    return {
+      askingAbout: null,
+      showAsk: false
+    }
+  },
+  methods: {
+    askAbout (e) {
+      this.askingAbout = e
+      this.showAsk = true
+    },
+    del (e) {
+      this.$emit('delete', e)
+      this.showAsk = false
+    }
+  }
+}
+</script>
+
+<template>
+  <el-menu :router="true">
+    <el-dialog
```

Comment:
Turns out there's another modal-like component that's meant for this purpose: The [Message Box](http://element.eleme.io/#/en-US/component/message-box).